src/ray/core_worker/core_worker.h

```diff
@@ -38,6 +38,10 @@ class CoreWorker {
   /// Language of this worker.
   Language GetLanguage() const { return language_; }
 
+  WorkerContext &Context() { return worker_context_; }
+
+  RayletClient &Raylet() { return *raylet_client_; }
```

Comment:
is this used?

Reply:
If this is used, better name it `GetRayletClient`

Reply:
Yes, it is used to let Java worker access raylet for things like checkpoint or dynamic resource. About naming, I followed other functions like `Tasks` and `Objects`. Maybe change them too?

Reply:
`Tasks` and `Objects` are a bit different. The naming convention is to use a plural word for util class about something. For example, `core_worker.Objects()` returns a util class that that contains all object-related methods. 
I think we can keep `Tasks` and `Objects`, and rename `Context` and `Raylet` to `GetWorkerContext` and `GetRayaletClient`

Reply:
Done.